src/main/java/com/rocketden/main/service/LiveGameService.java

```diff
@@ -0,0 +1,16 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.dao.RoomRepository;
+import org.springframework.stereotype.Service;
+
+/**
+ * Class to handle code updates and miscellaneous requests.
+ */
+@Service
+public abstract class LiveGameService extends GameManagementService {
+
+    protected LiveGameService(RoomRepository repository, SocketService socketService) {
+        super(repository, socketService);
+    }
```

Comment:
I'm not sure if this should be abstract/inherit from GameManagementService? I can see it potentially inheriting from like some generic GameService interface (though not sure what shared methods there'd be), but it seems like it has a pretty distinct function from GameManagementService

Reply:
Also, this is more of a general question that we might want to look into later but I'm not too familiar with the way inheritance works with Spring autowired annotations

Reply:
Yeah you're right, I think I made some poor design decisions here -- namely, I thought you could instantiate an abstract class and call the methods that are implemented, but that doesn't seem to be the case. If that's true, then I'll need to either implement all methods somehow (???) or, what's more likely, just simply extend the classes and not include any of the abstract methods.

Reply:
Perhaps I thought about the design wrong, as this would mean that I could accomplish the same thing by just not extending anything, and having them as separate service classes.

Reply:
I've changed this from abstract classes to autowiring now.